src/ngx_http_lua_util.c

```diff
@@ -371,40 +371,29 @@ ngx_http_lua_del_thread(ngx_http_request_t *r, lua_State *L,
 u_char *
 ngx_http_lua_rebase_path(ngx_pool_t *pool, u_char *src, size_t len)
 {
-    u_char            *p, *dst;
-
-    if (len == 0) {
+    ngx_str_t nsrc;
```

Comment:
`nsrc` is not a good name since the `n` prefix reminds me of `number`. But it's actually a string.